src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -187,6 +193,11 @@ private WebAssemblyHostEnvironment InitializeEnvironment(IJSUnmarshalledRuntime
         /// </summary>
         public ILoggingBuilder Logging { get; }
 
+        /// <summary>
+        /// Gets an object that holds options for allowing JavaScript to add root components dynamically.
+        /// </summary>
+        public DynamicRootComponentConfiguration DynamicRootComponents => _dynamicRootComponentConfiguration;
```

Comment:
```suggestion
        public JSRootComponentConfiguration JSRootComponents => _dynamicRootComponentConfiguration;
```

Reply:
```C#
public sealed class JSRootComponentConfiguration
{
    void Allow<TComponent>(string name);
}
```

```C#
host.RootComponents.Add<App>("#app");
builder.JSC.Allow<Counter>("counter");
```

Reply:
We came up with a different design later.